### PR TITLE
Feature/sort

### PR DESCRIPTION
**Pull request by markbolier**

### CODE REVIEW

File: src/components/team-entries/TeamEntries.tsx [hunk 1]
```diff
@@ -1,37 +1,57 @@
...
+        </Styled.Sort>
+        <Filter setCurrentClient={setCurrentClient} />
+      </Styled.Wrapper>
+      {orderedEntries
         .filter((teamEntry) => currentClient === "" || teamEntry.client === currentClient)
         .map((teamEntry, i, arr) => {
-          const currentClient = teamEntry.client;
-          const previousClient = arr[i - 1]?.client;
           return (
             <div key={Math.random()}>
```
Comment (amijsb) [inline src/components/team-entries/TeamEntries.tsx:54]: You could (and probably should) use `teamEntry.id` as a key.

File: src/components/time-entries/TimeEntries.tsx [hunk 2]
```diff
@@ -15,6 +15,27 @@ interface TimeEntriesProps {
 
 export const TimeEntries = ({ isModalActive, handleModal }: TimeEntriesProps) => {
   const { timeEntries, setTimeEntries } = useContext(TimeEntriesContext);
+  const [orderedEntries, setOrderedEntries] = useState(timeEntries);
+  const [ascending, setAscending] = useState(false);
+
+  const handleSort = async () => {
```
Comment (amijsb) [inline src/components/time-entries/TimeEntries.tsx:21]: I don't think this function needs to be async.

**Review by amijsb — CHANGES_REQUESTED** [review]
Nice! Good busy 🚀

File: src/components/time-entries/TimeEntries.tsx [hunk 3]
```diff
@@ -1,20 +1,41 @@
...
-export const TimeEntries = ({ isModalActive, handleModal }: Homepage) => {
-  const { timeEntries, setTimeEntries } = useContext(EntriesContext);
+export const TimeEntries = ({ isModalActive, handleModal }: TimeEntriesProps) => {
+  const { timeEntries, setTimeEntries } = useContext(TimeEntriesContext);
+  const [orderedEntries, setOrderedEntries] = useState(timeEntries);
+  const [ascending, setAscending] = useState(false);
+
+  const handleSort = async () => {
+    const sortedEntries = timeEntries.sort((a, b) => {
+      const startTimeA = new Date(a.startTime).valueOf();
```
[the remark this answers is not in the document]
Comment (markbolier) [reply src/components/time-entries/TimeEntries.tsx:23]: You're right! Good one :)

**Comment by sabinewst** [comment]
Unrelated to this PR, but your menu isn't currently accessible. You should be able to navigate to the various menu items with tab. I think if you add `a` tags it will be fine, check out the example here: https://nextjs.org/docs/api-reference/next/link

**Review by amijsb — APPROVED** [review]
💯

**Pull request merged**